frontend/src/views/landing-page/LandingPage.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { connect } from "react-redux";
+
+import Login from "../../components/Login.jsx";
+import SignUp from "../../components/SignUp.jsx";
+
+import proxy from "../../proxy.js";
+
+class UnconnectedLandingPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { username: "", password: "" };
+  }
+  submitLogIn = event => {
+    event.preventDefault();
+    let data = new FormData();
+    data.append("username", this.state.username);
+    data.append("password", this.state.password);
+    console.log(this.state.username + this.state.password);
+    console.log(event);
+    fetch(proxy + "/login", { method: "POST", body: data });
+  };
+  OnChangeUsername = event => {
+    this.setState({ username: event.target.value });
+  };
+  OnChangePassword = event => {
+    this.setState({ password: event.target.value });
+  };
+  handleLogOutClick = () => {
+    console.log("logging out");
+    document.cookie = "sid=undefined";
+    this.props.dispatch({ type: "logout" });
+  };
+  renderLoginSignUp = loggedIn => {
+    if (loggedIn !== false) {
+      return (
+        <div>
+          Logged In
+          <button onClick={this.handleLogOutClick}>Log out</button>
+        </div>
+      );
+    }
+    return (
+      <div>
```

Comment:
Nit: If you just need a single top-level component to wrap multiple children, you can use `React.Fragment`:

```
<React.Fragment>
  <Login />
  <SignUp />
</React.Fragment>
```

this has a shorthand:

```
<>
  <Login />
  <SignUp />
</>
```

In addition to being marginally less typing, this has the big benefit of not adding an extra layer in your HTML. This can be really important for things like CSS grid, flexbox, tables - an extra <div> between parent and child can break a lot of layout stuff, but fragments don't add any other markup.